configuration/wireless/access-point-routed.md

```diff
@@ -0,0 +1,190 @@
+# Setting up a Raspberry Pi as a Routed Wireless Access Point
+
+The Raspberry Pi can be used as a wireless access point routing to an existing ethernet network. This will create a new wireless network entirely managed by the Raspberry Pi.
```

Comment:
I'm not sure if the average reader will understand "a wireless access point routing to an existing ethernet network" - perhaps it could be clarified slightly? :man_shrugging: 

Although the IP-address diagram below definitely helps.

Reply:
I've been struggling with that... The current README.md file uses this expression: "[A guide to setting the Raspberry Pi up] as an access point, and as an internet sharing device"

On retrospect I think I should have used that.
"The Raspberry Pi can be used as a wireless access point, and as an internet sharing device. This will create ..."?

Reply:
I'm not sure that works either, since both pages talk about "wireless access point" and both pages end up sharing the internet to the wireless clients?
Would phrasing like "extending an existing network" and "creating a new sub-network" work? :man_shrugging:

Reply:
"extending an existing network" works for me for bridged
 "creating a new sub-network" not as much for routed, it sounds too much as "net/supernet/subnet" which have a definite meaning.
What about "secondary network" for routed?